src/components/SideNavigation.vue

```diff
@@ -9,15 +9,15 @@
           </button>
         </template>
         <template v-slot:items class="font-circular">
-          <our-dropdown-item class="flex justify-between items-center">New Accomplishment</our-dropdown-item>
-          <our-dropdown-item class="flex justify-between items-center">New Update</our-dropdown-item>
-          <our-dropdown-divider />
-          <our-dropdown-item>New Job Post</our-dropdown-item>
+          <!-- <our-dropdown-item class="flex justify-between items-center">New Accomplishment</our-dropdown-item> -->
```

Comment:
you disabled thses. is that by design?

Reply:
Those were menu items that were added in the design but we don't have any functionality for them. So I opted to remove them rather than leave them there

Reply:
alright 👍